flink-streaming-scala/src/test/scala/org/apache/flink/streaming/api/scala/WindowFoldITCase.scala

```diff
@@ -70,7 +70,7 @@ class WindowFoldITCase extends StreamingMultipleProgramsTestBase {
     source1
       .keyBy(0)
       .window(TumblingEventTimeWindows.of(Time.of(3, TimeUnit.MILLISECONDS)))
-      .fold(("R:", 0), { (acc: (String, Int), v: (String, Int)) => (acc._1 + v._1, acc._2 + v._2) })
+      .fold(("R:", 0)){ (acc: (String, Int), v: (String, Int)) => (acc._1 + v._1, acc._2 + v._2) }
```

Comment:
I have no strong preference here. I didn't introduce them initially because you also don't insert a space between a function name and the first parameter list. But if you think that it's better, I'll change it.